Valeriia/Movies_database_project/src/store/sagas/moviesSagas.ts

```diff
@@ -0,0 +1,28 @@
+import {MoviesActionTypes} from '../actions/actionTypes';
+import {all, call, put, takeLatest} from 'redux-saga/effects';
+import {
+    IRequestMoviesAction,
+    onRequestErrorMovies, onRequestSuccessMovies,
+} from '../actions/moviesAction';
+import {fetchListOfMoviesBySortType} from './services';
+
+function* requestMoviesSaga(action: IRequestMoviesAction){
+    try{
+        const movies = yield call(fetchListOfMoviesBySortType, action.sortByType, action.searchBy, action.searchValue );
```

Comment:
fetchListOfMoviesBySortType - kind of confusing name, because you are fetching not only by sortType.

optional: rename please